### ISSUE TITLE
Missing PDF?

### ISSUE THREAD

**Issue by robjwood** [UKHomeOffice/posters#47]
Using the git-browser page, it looks like the _accessibility-posters-next trial.pdf_ is missing. I get a 404 when I click on the thumbnail below.

![image](https://user-images.githubusercontent.com/1257421/27484473-3f5fabee-5821-11e7-81a9-6d275fb9e7cb.png)

**Comment by joelanman** [comment]
sorry - my fault, Git Browser is a separate project (https://github.com/joelanman/git-browser) and I'll update it now

**Comment by joelanman** [comment]
should be ok now - thanks for raising!

**Comment by robjwood** [comment]
No problem, it's a great resource!

**Comment by joelanman** [comment]
thanks! Do you mind closing this issue now?

**Comment by robjwood** [comment]
Oops! Of course.
